nbs/tutorials/extensions.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>I'm not a fan of claiming something is exceedingly easy or trivial up front, for similar reasons to this:</p><p> </p><ul><li>https://www.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/ and</li><li>https://justsimply.dev/</li></ul><p> </p><p> </p><p>I actually don't understand what the <code>---</code> annotation refers to that you mention here?  What does that. mean?   </p><p> </p><p>Consider linking to the directives page, as that is nbdev specific terminology. </p><p> </p><p>Also, is this article about writing a custom "plugin" or is it about extending nbdev to add your own directives?  I think the word "plugin" assumes a much broader scope and implies something different.  IIUC the thing we are trying to tackle in this article is adding your own custom directives?   Being clearer about the scope might make the article less confusing in this regard.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>In regards to plugin, writing a plugin and extending the nbdev directives mean the same thing when I consider an extension hence why they're the same here</p>